src/Raven.Client/Documents/Subscriptions/Subscription.cs

```diff
@@ -237,6 +240,10 @@ public Task StartAsync()
             return tcs.Task;
         }
 
+        private ServerNode _redirectNode;
+
+        public string CurrentNodeTag => _redirectNode.ClusterTag;
+
         private async Task<Stream> ConnectToServer()
```

Comment:
We need to do a summary of the interaction between server & client for subscription. I want to have it all in one place, so we can trace how it behave and compare it to the code.